src/js/view/components/menu.js

```diff
@@ -2,34 +2,51 @@ define([
     'view/components/tooltip',
     'utils/helpers',
     'utils/underscore',
+    'events/events',
+    'utils/ui',
     'handlebars-loader!templates/menu.html'
-], function(Tooltip, utils, _, menuTemplate) {
+], function(Tooltip, utils, _, events, UI, menuTemplate) {
 
     var Menu = Tooltip.extend({
+        'constructor' : function(name) {
+            Tooltip.call(this, name);
+            this.iconUI = new UI(this.el).on(events.touchEvents.TAP, utils.noop);
```

Comment:
Why extend the constructor, if all of this can go into the setup method?

It seems like since sometimes these will never be set up at all (for example a CC button when there are no closed captions), it would be a waste of effort to set up all the UI events.